src/cmd.ts

```diff
@@ -192,11 +192,7 @@ function createCommandWithSharedOptions(name: string, description: string) {
 			"--season-from-episodes <decimal>",
 			"Match season packs from episode torrents",
 			parseFloat,
-			fallback(fileConfig.seasonFromEpisodes, 1),
-		)
-		.option(
-			"--no-season-from-episodes",
-			"Don't match season packs from episode torrents",
+			fallback(fileConfig.seasonFromEpisodes, null),
 		)
```

Comment:
we should keep this for overriding the file config

Reply:
Yep, I just realized this myself.